### DISCUSSION TITLE
Weighted plots gives a chance to us early farmers

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#5971] [Ideas]
Hi,

Since netspace keeps growing, early farmers are losing out as time goes, and farming seems less and less interesting.

### The idea

Give a **weight** to early plotted farms, the earlier a plot has been added to the blockchain the more chances it should get at winning some coins. One weight point can be given per week, so the earliest plots will have the least points therefore the most chances.

### Example

Week 1 plots get 1 point, week 8 plots get 8 points, and you can use weighting to assign higher probability of winning to plots with lower points.

This makes it fair for those who started earlier, plotted tons, and are limited to farming only.

I don't fully understand how this blockchain/chia system works, so I don't know if this idea is even feasible.

**Comment by mariano54** [comment]
It won't work, the blockchain does not care how old a plot is, all plots are the same. 
However the rewards are already favoring early participants. In the first 3 years, there are 2XCH per block, this halves at 3 years, 6 years, and 9 years. 
Also the netspace is likely to never be as low as it is now.

**Reply by ghost** [reply]
Oh I see, thank you for the quick response.
